packages/ag-charts-community/src/dom/focusIndicator.ts

```diff
@@ -23,10 +23,28 @@ export class FocusIndicator {
         this.swapChain.addListener('swap', (parent) => this.onSwap(parent));
     }
 
-    updateBounds(bounds: Path | BBoxValues | undefined, hoverRect?: { x: number; y: number }) {
+    updateBounds(bounds: Path | BBox | undefined, hoverRect?: BBox) {
         if (bounds === undefined) {
-            // skip
+            return;
+        }
+
+        let boundsVisible: boolean;
+        if (hoverRect == null || bounds == null) {
+            boundsVisible = true;
         } else if (bounds instanceof Path) {
+            boundsVisible = hoverRect.intersection(bounds.getBBox()) != null;
+        } else {
+            boundsVisible = hoverRect.intersection(bounds) != null;
+        }
+
+        if (!boundsVisible) {
+            this.element.style.display = 'none';
```

Comment:
Do not touch the `display` property. The WCAG success criteria requires the focus indicator to remain visible at all times for keyboard users. If the input `bounds` does not intersect with the `hoverRect`, then it's much better to draw a focus indicator with a height or width of 0 than to draw nothing at all.